db/1.8/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -738,6 +738,16 @@ Invokes a stored procedure on the database.  When the stored procedure returns o
 === For Configurations
 * <<config>>
 
+=== Working with Pooling Profiles
+
+The Stored procedure operation closes the connections it receives once the flow containing the operation completes. This behavior occurs because the resultset the operation generates can have a stream or be part of an ongoing transaction. Thereby, Mule runtime engine never knows when the flow is going to consume the resultset in its entirety nor when the connection is going to be needed.
+
+Starting on Database Connector v1.8.3 the connections on the Stored procedure operation are released if they are not part of a stream nor a transaction.
+
+When executing the Stored procedure operation with a pooling profile consider that because connections do not release until the flow completes, the number of available connections should be equal or greater than the ones that are going to be used.
```

Comment:
```suggestion
When executing the *Stored procedure* operation with a pooling profile, consider that because connections do not release until the flow completes, the number of available connections must be equal to, or greater than, the connections that will be used.
```